test/clientTest.js

```diff
@@ -153,15 +151,14 @@ mc.supportedVersions.forEach(function (supportedVersion, i) {
         })
       })
 
-      it('produce a decent error when connecting with the wrong version', function (done) {
+      it('Produces a decent error when connecting with the wrong version', function (done) {
         const client = mc.createClient({
           username: 'Player',
           version: version.minecraftVersion === '1.8.8' ? '1.11.2' : '1.8.8',
           port: PORT
         })
         client.once('error', function (err) {
           if (err.message.startsWith('This server is version')) {
-            console.log(new Date() + 'Correctly got an error for wrong version : ' + err.message)
```

Comment:
Don't remove this

Reply:
Why not? In any case we should log something when we get an unexpected error, we shouldn't log "everything is going ok" messages

Reply:
Not convinced. I only recently fixed this test and this message was very useful to know there was no problem with the test and actually there was a problem with some other unrelated error happening here.